src/quo2/components/list_item.cljs

```diff
@@ -0,0 +1,250 @@
+     (if (string? title)
+       [text/text {:weight                    (or title-text-weight :regular)
+                   :number-of-lines           1
+                   :style                     {:color text-color}
+                   :title-accessibility-label title-accessibility-label
+                   :ellipsize-mode            :tail
+                   :size                      (or text-size (size->single-title-size size))}
+        title]
+       title))])
+
+(defn left-side [props]
+  [rn/view {:style {:flex-direction :row
+                    ;; Occupy only content width, never grow, but shrink if need be
+                    :flex-grow      0
+                    :flex-shrink    1
+                    :padding-right  16
+                    :align-items    (or (:left-side-alignment props) :center)}}
+   [icon-column props]
+   [title-column props]])
+
+(defn right-side [{:keys [chevron active disabled accessory accessory-text accessory-style animated-accessory?]}]
+  (when (or chevron accessory)
+    [rn/view {:style (merge {:align-items     :center
+                             :justify-content :flex-end
+                             :flex-direction  :row
+                             ;; Grow to occupy full space, shrink when need be, but always maitaining 16px left gutter
+                             :flex-grow       1
+                             :flex-shrink     0
+                             :margin-left     16
+                             ;; When the left-side leaves no room for right-side, the rendered element is pushed out. A flex-basis ensures that there is some room reserved.
+                             ;; The number 80px was determined by trial and error.
+                             :flex-basis      80}
+                            accessory-style)}
+     [rn/view {:style (:tiny spacing/padding-horizontal)}
+      (case accessory
+        :radio    [controls/radio {:value active :disabled disabled}]
+        :checkbox [(if animated-accessory?
+                     controls/animated-checkbox
+                     controls/checkbox)
+                   {:value active :disabled disabled}]
+        :switch   [controls/switch {:value active :disabled disabled}]
+        :text     [text/text {:color           :secondary
+                              :ellipsize-mode  :middle
+                              :number-of-lines 1}
+                   accessory-text]
+        accessory)]
+     (when (and chevron platform/ios?)
+       [rn/view {:style {:padding-right (:tiny spacing/spacing)}}
+        [icons/icon :main-icons/next {:container-style {:opacity         0.4
+                                                        :align-items     :center
+                                                        :justify-content :center}
+                                      :resize-mode     :center
+                                      :color           (theme-colors colors/neutral-50 colors/neutral-50)}]])]))
+
+(defn list-item
```

Comment:
if its a copy, why ?  better to just use quo for now

Reply:
It has some differences on icon sizing, text, spacing, theming, etc.

Reply:
They are not exactly the same, it has many styling changes to comply with Figma designs.
We could use quo version but it will not match designs at all